src/main/java/com/capgemini/fs/coindashboard/controller/exceptionHandler/ApiErrorController.java

```diff
@@ -0,0 +1,18 @@
+package com.capgemini.fs.coindashboard.controller.exceptionHandler;
+
+import lombok.extern.log4j.Log4j2;
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Log4j2
+@Controller
+public class ApiErrorController implements ErrorController {
```

Comment:
Can we get rids of this ApiErrorController and do error handling in ControllerAdvice (some base controller advice), in the controller advice we can extends ResponseEntityExceptionHandler class from Spring